runtime/bindings/python/src/pyopenvino/graph/variant.hpp

```diff
@@ -20,48 +20,50 @@ template <typename VT>
 extern void regclass_graph_VariantWrapper(py::module m, std::string typestring)
 {
     auto pyclass_name = py::detail::c_str((std::string("Variant") + typestring));
```

Comment:
This should not be required anymore as `ov::Any` is not templated like `Variant` was before. It may require to rewrite a whole file here.

Please correct me if I am wrong.

Reply:
Thus I'm not going to change PyAPI in this review. But the issue https://github.com/pybind/pybind11/issues/1590 says that there is no way to make universal `any` bind without specialization for holder type.  So we have to register some special classes here.